modules/tinymce/src/core/test/ts/browser/html/StylesTest.ts

```diff
@@ -234,4 +237,124 @@ describe('browser.tinymce.core.html.StylesTest', () => {
     assertStyles(styles, 'background:url(javascript:alert(1)', `background: url('javascript:alert(1');`);
     assertStyles(styles, 'background:url(vbscript:alert(1)', `background: url('vbscript:alert(1');`);
   });
+
+  const applyAndAssertForeColor = (editor: Editor, appliedColor: string, expectedColor: string) => {
```

Comment:
I think all these tests should be moved to a new test file for that particular option maybe a `ForceHexColorTest.ts` file. Since these tests do use a full editor so they are more kind of higher up integration tests of testing the option and the feature at an editor level.

However this file should also have tests but specific to that module the so tests that would run for example `Styles({ force_hex_color: 'always' }` and then assert that the parse and serialize functions properly convert the colors. So a few `it` test for each of the option cases. This way if you work on that part of the code you could easily test things in isolation from the editor and more easily debug things as well since the code path would be more shallow.

- [ ] Split the tests and add some module specific ones

Reply:
I've moved the Editor UI related tests to their own file in `browser/editor/ForceHexColorTest.ts`. Let me know if there's a better spot to put in.
Also added in some tests for `Styles({ force_hex_color: '...' })`